resource-watch/prep-app
# NexGDDP/Loca: let the user copy and paste the map position

This PR lets the user copy the position of a NexGDDP/Loca map to apply it to another one.

<p align="center">
<img width="588" alt="Tooltip on the map showing the zoom level, the center coordinates and buttons to copy and paste them" src="https://user-images.githubusercontent.com/6073968/45353715-83371a00-b5b3-11e8-8851-ce0791849506.png">
<p>

Note that the button to open this tooltip has an icon really similar to the one used to position the marker. We should probably ask the designers to get a new one:

<p align="center">
<img width="153" alt="Buttons on the map" src="https://user-images.githubusercontent.com/6073968/45353909-2b4ce300-b5b4-11e8-9cfa-b1e9661b7623.png">
</p>

## Testing instructions
1. Go to this dataset (Loca) http://localhost:5000/dataset/loca_dry_rcp45_decadal
2. Move the map, click the new button and copy the position
3. Go to this other dataset (NexGDDP) http://localhost:5000/dataset/cum_pr_rcp45_decadal
4. Click the new button and then the paste button

The map should have the same position and same zoom as the map on the first dataset.

## Pivotal

[Task](https://www.pivotaltracker.com/story/show/160143251)
